frontend/src/app/testResults/submissions/Submissions.tsx

```diff
@@ -0,0 +1,176 @@
+import { useParams } from "react-router-dom";
+import moment from "moment";
+import { DatePicker, Label } from "@trussworks/react-uswds";
+import { useState } from "react";
+
+import Pagination from "../../commonComponents/Pagination";
+import { formatDateWithTimeOption } from "../../utils/date";
+import {
+  GetUploadSubmissionsQuery,
+  useGetUploadSubmissionsQuery,
+} from "../../../generated/graphql";
+import {
+  // setEndTimeForDateRange,
+  setStartTimeForDateRange,
+} from "../../analytics/Analytics";
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+
+const Submissions = () => {
+  const urlParams = useParams();
+  const pageNumber = Number(urlParams.pageNumber) || 1;
+  const pageSize = 10;
+
+  const [startDate, setStartDate] = useState<string | null>(null);
+  const [endDate, setEndDate] = useState<string | null>(null);
+  const [resetCount, setResetCount] = useState<number>(0);
+
+  const { data: submissions, loading, error } = useGetUploadSubmissionsQuery({
+    fetchPolicy: "no-cache",
+    variables: {
+      pageSize,
+      pageNumber: pageNumber - 1,
+      startDate,
+      endDate,
+    },
+  });
+
+  console.log(submissions);
```

Comment:
Can you take out this debugging statement?

Reply:
lol thanks for catching this!